exporter/datasetexporter/examples/e2e/otelcol-builder.yaml

```diff
@@ -0,0 +1,39 @@
+# List of all components is here:
```

Comment:
This is going to be hard to maintain, any chance we can simplify this example?

Reply:
A maintainer would be able to offer a more decisive take - if you list components and we need to update them as part of the release, it will be an additional maintenance cost.

Reply:
You don't need the builder configuration file to test the example config (the Datadog exporter is only testing this on unit tests, not by running a Collector). What benefit does the builder config file bring to the example?